pf-keys/definitions/geom.yaml

```diff
@@ -1480,6 +1488,151 @@ TopoSlopesY:
         domains:
           DoubleValue:
 
+# -----------------------------------------------------------------------------
+# ChannelWidth
+# -----------------------------------------------------------------------------
+# ChannelWidth missing from manual and grep
+
+# TODO: Change the documentation to match with channel width, also add handlers if needed? (Not sure if they are needed)
+
+ChannelWidth:
+  __doc__: >
+    Setting filename for channel width data 
+  Width:
+    __doc__: >
+      Setting filename for channel width data
+    FileName:
+      help: >
+        [Type: string] This key is the name of the PFB file that contains channel width data which ParFlow uses.
+      domains:
+        Anystring:
+        ValidFile:
+
+ChannelWidthX:
+  __doc__: >
+    Setting data for channel width in the X direction
+  GeomNames:
+    help: >
+      [Type: string] This key specifies all of the geometries on which different channel width x values will be assigned. Channel widths
+      may be assigned by PFBFile or as Constant by geometry. These geometries must cover the entire upper surface of the
+      computational domain. 
+    domains:
+      EnumDomain:
+        locations: 
+          - /GeomInput/{GeomInputItem}/GeomNames
+          - /GeomInput/{GeomInputItem}/GeomName
+    handlers:
+      ChannelWidthXUpdater:
+        type: ChildrenHandler
+        class_name: ChannelWidthXItem
+        location: ./Geom
+
+  # in manual, need to consolidate duplicates (manual also includes NetCDF file capability, which is included here)
+  Type:
+    help: >
+      [Type: string] This key specifies which method is to be used to assign channel widths. The choices currently available are
+      Constant which indicates that a constant is to be assigned to all grid cells within a geometry and PFBFile which
+      indicates that all values are read in from a distributed, grid-based ParFlow binary file. If NetCDF is used, NCFile can be
```

Comment:
Is NetCDF reading slopes here?  Is this a cut-paste error?

Reply:
Looks like cut-paste error from TopoSlopes.